rust/agents/validator/src/validator.rs

```diff
@@ -204,6 +205,27 @@ impl Validator {
         tasks
     }
 
+    fn log_on_announce_failure(result: ChainResult<TxOutcome>) {
+        match result {
+            Ok(outcome) => {
+                if !outcome.executed {
+                    error!(
+                        hash=?outcome.txid,
+                        gas_used=?outcome.gas_used,
+                        gas_price=?outcome.gas_price,
+                        "Transaction attempting to announce validator reverted. Make sure you have enough ETH in your account to pay for gas."
+                    );
+                }
+            }
+            Err(e) => {
+                error!(
+                    error=?e,
```

Comment:
Pretty sure we were standardizing on `err`.
```suggestion
            Err(err) => {
                error!(
                    ?err,
```